lib/dgram.js

```diff
@@ -543,7 +563,7 @@ Socket.prototype.addMembership = function(multicastAddress,
   this._healthCheck();
 
   if (!multicastAddress) {
-    throw new Error('multicast address must be specified');
+    throw new errors.Error('ERR_MISSING_ARGS', 'multicastAaddress');
```

Comment:
I think as long as we are consistent across the board, then I'm ok based on the example set by v8